spec/integration/retryable_writes_errors_spec.rb

```diff
@@ -31,4 +31,61 @@
       end
     end
   end
+
+  context "when encountering a WriteConcernError with a RetryableWriteError label" do
```

Comment:
Shouldn't this say something about NoWritesPerformed?

Reply:
done